src/appleseed/foundation/meta/tests/test_imagefilewriter.cpp

```diff
@@ -0,0 +1,219 @@
+// appleseed.foundation headers.
+#include "foundation/image/color.h"
+#include "foundation/image/colorspace.h"
+#include "foundation/image/genericimagefilereader.h"
+#include "foundation/image/genericimagefilewriter.h"
+#include "foundation/image/image.h"
+#include "foundation/image/imageattributes.h"
+#include "foundation/image/pixel.h"
+#include "foundation/math/scalar.h"
+#include "foundation/utility/iostreamop.h"
+#include "foundation/utility/test.h"
+
+// Standard headers.
+#include <cassert>
+#include <cmath>
+#include <cstddef>
+#include <cstring>
+#include <exception>
+#include <memory>
+
+using namespace foundation;
+
+/*
+// Default export formats:
```

Comment:
Comments lack of context, you can explain what is the global goal of the test file here, and what each column of your table contain

Reply:
Or remove this comment block and comment each test, as I did in the example I sent you.